config/routes.rb

```diff
@@ -42,8 +42,12 @@
         get(:healthcheck)
       end
     end
-    resource :pension_claims, only: [:create]
-    resource :burial_claims, only: [:create]
+
+    # Disabled in production until Central Mail API is in place
+    unless Rails.env.production?
```

Comment:
can you change this to use `Settings.pension_burial.upload.enabled`? staging and dev are both run with `RAILS_ENV=production` and we want to be able to test the central mail API there

Reply:
Ah ok. Makes sense. Thought they'd be running under dev or test config. I'll fix that.